packages/nouns-contracts/abi/contracts/governance/NounsDAOLogicV3.sol/NounsDAOLogicV3.json

```diff
@@ -44,30 +44,6 @@
     "name": "UnsafeUint16Cast",
     "type": "error"
   },
-  {
-    "inputs": [],
-    "name": "VotesBelowProposalThreshold",
```

Comment:
@eladmallel there are a lot of changes in this file, do they all make sense?
maybe we should go over them